packages/shared/lib/core/wallet/utils/generateActivity/helper/getSendingInformation.ts

```diff
@@ -1,7 +1,7 @@
 import { IAccountState } from '@core/account'
-import { ActivityDirection } from '@core/wallet/enums'
-import { IProcessedTransaction } from '@core/wallet/interfaces'
-import { Output, Subject } from '@core/wallet/types'
+import { ActivityDirection } from '../../../enums'
+import { IProcessedTransaction } from '../../../interfaces'
+import { Output, SenderInfo } from '../../../types'
```

Comment:
i would revert this change and once we agree on our approach to avoid circular dependencies, do it everywhere in one go